### PR TITLE
Create BitKong.com.fc

### PR DESCRIPTION
**Pull request by aviramdayan**
create

### CODE REVIEW

File: BitKong.com/BitKong.com.fc [hunk 1]
```diff
@@ -0,0 +1,673 @@
...
+//============================================================//
+//the number of "successes" in the script - when are finished  
+//============================================================//
+    public override void Start()
+    {
+        //Title that shows in the browser. Is used to identify and close popup windows
+        Title = "BitKong.com";
+
+        //After we did try to claim on the faucet we search for these elements to determine if it was a success or a fail
+		SuccessXPath = "//button[contains(@class,'action-button sc-bZQynM eVpSUG')]"; //*[text()='']
```
Comment (faucetcollector) [inline BitKong.com/BitKong.com.fc:69]: these classes sound like auto generated, so they are likely to change when they update the website

File: BitKong.com/BitKong.com.fc [hunk 2]
```diff
@@ -0,0 +1,673 @@
...
+     GoToUrl("https://bitkong.com/en/home");
+	 }
+	 
+     //save Login Cookies - Normal- LOGOUT
+     return ElementByXPath("//div[@class='username']") != null;  		
+		
+		
+     if (!Driver.Url.Contains("https://www.facebook.com/"))
+     {
+     GoToUrl("https://www.facebook.com/");
```
Comment (faucetcollector) [inline BitKong.com/BitKong.com.fc:121]: why does it need to go to facebook?

File: BitKong.com/BitKong.com.fc (hunk 1 above)
Comment (aviramdayan) [reply BitKong.com/BitKong.com.fc:69]: yes and no, this is the only option for now...

File: BitKong.com/BitKong.com.fc (hunk 2 above)
Comment (aviramdayan) [reply BitKong.com/BitKong.com.fc:121]: if user want to login with facebook, to save the cookies

**Review by faucetcollector — APPROVED** [review]

**Pull request merged**